_component_design/tables-controls.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Tables - Controls
```

Comment:
file name can stay the same, but the title should be "Table Controls"

Reply:
updated.